HashCopy/Public/Copy-FileHash.ps1

```diff
@@ -110,7 +116,13 @@ function Copy-FileHash {
     }
     Process {
         ForEach ($Source in $SourcePath) {
-            $SourceFiles = (Get-ChildItem -Path $Source -Recurse:$Recurse -File).FullName

```

Comment:
It looks like its valid for `-Exclude` to be passed `$null` so we can just always include it here and it negates the need for the rest of the additions.

Reply:
No there's no need to convert it to an array, because we've made the input parameter an array.